tools/cldr-code/src/main/java/org/unicode/cldr/util/PathDescription.java

```diff
@@ -382,10 +382,18 @@
         + RegexLookup.SEPARATOR
```

Comment:
I think we also need to do the PlaceHolderDescriptions (but not in this PR). I think our tests might ignore placeholders that aren't {0}, {1}, etc.

Reply:
I will add that to the TODOs in the agenda

Reply:
SGTM